assets/css/styles.css

```diff
@@ -0,0 +1,518 @@
+* {
+  font-family: "Open Sans", sans-serif;
+}
+
+body {
+  margin: 0;
+  background: url("../../images/desert_photo.jpg") no-repeat center center fixed;
+  background-size: cover;
+}
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+#header-logo {
+  height: 10rem;
+  width: 10rem;
+  border: 4px solid #fff;
+  border-radius: 50%;
+}
+
+div#hero-image-container {
+  display: flex;
+  align-items: center;
+  margin: 2%;

```

Comment:
Don't use percentages for margins. You can `rem` or `px` but `%` is hard to control in margins.